extensions/ext/GL_EXT_shader_realtime_clock.txt

```diff
@@ -40,6 +41,9 @@ Dependencies
 
     This extension interacts with GL_ARB_gpu_shader_int64.
 
+    Additional changes are written against the OpenGL ES Shading
+    Language Specification, Version 3.10.4, dated 2016/01/29.
+
```

Comment:
I think we would normally write something like:
```suggestion
    This extension requires OpenGL 4.4 or OpenGL ES 3.1.

```
(I copied this from https://registry.khronos.org/OpenGL/extensions/KHR/KHR_shader_subgroup.txt which is defined for both GL and ES.)